src/parallel_parameter_search/walk_optimization.py

```diff
@@ -66,21 +66,37 @@ def suggest_walk_params(self, trial):
     def correct_pitch(self, x, y, yaw):
         return self.trunk_pitch + self.trunk_pitch_p_coef_forward * x + self.trunk_pitch_p_coef_turn * yaw
 
-    def evaluate_direction(self, x, y, yaw, iteration, time_limit, start_speed=True):
+    def evaluate_direction(self, x, y, yaw, time_limit, start_speed=True):
         if time_limit == 0:
             time_limit = 1
         if start_speed:
             # start robot slowly
-            self.set_cmd_vel(x * iteration / 4, y * iteration / 4, yaw * iteration / 4)
+            self.set_cmd_vel(x / 4, y / 4, yaw / 4)
```

Comment:
This seems hacky, is there any reason for the /4?

Reply:
that's just to start the walking first with a quater of the speed as it will fall down otherwise due to high accelerations. It does not matter so much which exact speed factor is used. Also I don't have an idea on how this could be better implemented